app.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "GOV.UK Frontend Flask",
```

Comment:
Is any of this actually used?  Can this file be removed?

Reply:
Removed.